app/src/main/java/wishcantw/vocabulazy/audio/VLTextToSpeech.java

```diff
@@ -128,6 +135,14 @@ boolean isLanguageAvailable() {
         return (mTextToSpeech.isLanguageAvailable(Locale.ENGLISH) >= 0 && mTextToSpeech.isLanguageAvailable(Locale.TAIWAN) >= 0);
     }
 
+    public void mute() {
+        AppPreference.getInstance().setPlayerVolume(0.0f);
+    }
+
+    public void unMute() {
+        AppPreference.getInstance().setPlayerVolume(1.0f);
+    }
+
```

Comment:
FYI, the volume is being set as a parameter of Text-to-Speech, therefore it affects only within our app. And you may also ask, will it affect the voice in Exam? The answer is yes...